pkg/cmd/server/kube_master.go

```diff
@@ -44,6 +51,11 @@ func (cfg Config) BuildKubernetesMasterConfig(requestContextMapper kapi.RequestC
 		masterIP = addrs[0]
 	}
 
+	// in-order list of plug-ins that should intercept admission decisions
+	// TODO: add NamespaceExists
+	admissionControlPluginNames := []string{"LimitRanger", "ResourceQuota", "AlwaysAdmit"}
```

Comment:
Is AlwaysAdmit needed at the end of the chain? I thought admission plugins only gave errors, so if it made it to the end of the chain without an error, it was implicitly accepted